js/index.js

```diff
@@ -24,19 +35,66 @@ function calculateAll() {
 
 function removeProduct(event) {
   const target = event.currentTarget;
-  console.log('The target in remove is:', target);
+  const parent = target.parentNode.parentNode.parentNode;
+  parent.removeChild(target.parentNode.parentNode)
+  calculateAll()
+}
   //... your code goes here
+/*
+function testRow() {
+  const newRow = document.getElementById('cart');
+  newRow.insertRow(-1)
+  console.log(newRow)
 }
+testRow()*/
 
 // ITERATION 5
 
-function createProduct() {
-  //... your code goes here
+function createProduct(event) {
+  const targetName = document.querySelector('.create-product').querySelector('input[type="text"]')
+  let nameValue = targetName.value;
+  let priceInput = document.querySelector('.create-product').querySelector('input[type="number"]')
+  let priceValue = parseFloat(priceInput.value).toFixed(2)
+  console.log(nameValue + priceValue)
+  const target = event.currentTarget;
+  const parent = target.parentNode.parentNode.parentNode.parentNode;
+  const tableRows = document.getElementsByTagName('tbody')[0];
+  let newRow = tableRows.insertRow(-1)
+  newRow.innerHTML = `<tr class="product">
+  <td class="name">
+    <span>${nameValue}</span>
+  </td>
+  <td class="price">$<span>${priceValue}</span></td>
+  <td class="quantity">
+    <input type="number" value="0" min="0" placeholder="Quantity" />
+  </td>
+  <td class="subtotal">$<span>0</span></td>
+  <td class="action">
+    <button class="btn btn-remove">Remove</button>
+  </td>
+ 
+</tr>`;
+
+  console.log(tableRows);
+  //tableRows.classList.add('product')
+  calculateAll();
+  nameValue = ''
+  priceValue = ''
+  
 }
 
 window.addEventListener('load', () => {
   const calculatePricesBtn = document.getElementById('calculate');
   calculatePricesBtn.addEventListener('click', calculateAll);
 
+  const removeBtn = document.querySelectorAll('.btn.btn-remove');
```

Comment:
Since all the remove buttons are assigned to this variable, I would recommend using a plural name.